tests/unit/test_serialiation_error_checking.cc

```diff
@@ -0,0 +1,160 @@
+/*
+//@HEADER
+// *****************************************************************************
+//
+//                    test_serialiation_error_checking.cc
```

Comment:
I just noticed, there's a typo in the name of this file, and hence in the header

Reply:
Renamed.